vendor/github.com/opencontainers/runc/libcontainer/apparmor/apparmor.go

```diff
@@ -12,21 +13,17 @@ import (
 
 // IsEnabled returns true if apparmor is enabled for the host.
 func IsEnabled() bool {
-	if _, err := os.Stat("/sys/kernel/security/apparmor"); err == nil && os.Getenv("container") == "" {
-		if _, err = os.Stat("/sbin/apparmor_parser"); err == nil {
```

Comment:
This line might be related, I guess

Reply:
Not at my computer right now; would you be able to open a ticket in runc?

Reply:
I'm not sure we should have a ticket in runc repo, if the runc binary itself isn't breaking compatibility. runc basically doesn't care about compatibility of libcontainer Go pkgs.